### PR TITLE
[SchemaRegistry] prepare avro for release

### PR DESCRIPTION
**Pull request by swathipil**
update setup.py for release

### CODE REVIEW

File: sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Release History
 
-## 1.0.0b2 (Unreleased)
+## 1.0.0b2 (2020-08-17)
```
Comment (yunhaoling) [inline sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md:3]:
```suggestion
## 1.0.0b2 (2021-08-17)
```

File: sdk/schemaregistry/azure-schemaregistry-avroserializer/setup.py [hunk 2]
```diff
@@ -65,7 +65,7 @@
     zip_safe=False,
     packages=find_packages(exclude=exclude_packages),
     install_requires=[
-        'azure-schemaregistry==1.0.0b1',
+        'azure-schemaregistry==1.0.0b2',
```
Comment (yunhaoling) [inline sdk/schemaregistry/azure-schemaregistry-avroserializer/setup.py:68]: version.py also needs update to 1.0.0b2?

File: sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md (hunk 1 above)
Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md:3]: 🤦 thank you!

File: sdk/schemaregistry/azure-schemaregistry-avroserializer/setup.py (hunk 2 above)
Comment (swathipil) [reply sdk/schemaregistry/azure-schemaregistry-avroserializer/setup.py:68]: weirdly enough, that's already at 1.0.0b2? https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/schemaregistry/azure-schemaregistry-avroserializer/azure/schemaregistry/serializer/avroserializer/_version.py

**Review by yunhaoling — APPROVED** [review]
LGTM, pre-approval pending sr release and re-recording of the tests in sr avro serializer

**Pull request merged**